homeassistant/components/matter/strings.json

```
@@ -441,6 +441,30 @@
       "evse_soc": {
         "name": "State of charge"
       },
+      "operational_error": {
+        "name": "Operational error",
+        "state": {
+          "no_error": "No error",
+          "unable_to_start_or_resume": "Unable to start or resume",
+          "unable_to_complete_operation": "Unable to complete operation",
+          "command_invalid_in_state": "Command invalid in state",
```

Also a small thing: I had to look up the spec to see what "Command invalid in state" meant exactly.
What about adding "in **current** state"? That makes it a bit more clear to me at least, but it might be a bit too long...
```suggestion
          "command_invalid_in_state": "Command invalid in current state",
```
Any other ideas for this? Otherwise, "Command invalid in state" might be fine as well.